src/gigs-board/pages/Community.jsx

```diff
@@ -0,0 +1,204 @@
+/* INCLUDE: "common.jsx" */
+const nearDevGovGigsContractAccountId =
+  props.nearDevGovGigsContractAccountId || "devgovgigs.near".split("/", 1)[0];
+const nearDevGovGigsWidgetsAccountId =
+  props.nearDevGovGigsWidgetsAccountId ||
+  (context.widgetSrc ?? "devgovgigs.near").split("/", 1)[0];
+
+function widget(widgetName, widgetProps, key) {
+  widgetProps = {
+    ...widgetProps,
+    nearDevGovGigsContractAccountId: props.nearDevGovGigsContractAccountId,
+    nearDevGovGigsWidgetsAccountId: props.nearDevGovGigsWidgetsAccountId,
+    referral: props.referral,
+  };
+  return (
+    <Widget
+      src={`${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.${widgetName}`}
+      props={widgetProps}
+      key={key}
+    />
+  );
+}
+
+function href(widgetName, linkProps) {
+  linkProps = { ...linkProps };
+  if (props.nearDevGovGigsContractAccountId) {
+    linkProps.nearDevGovGigsContractAccountId =
+      props.nearDevGovGigsContractAccountId;
+  }
+  if (props.nearDevGovGigsWidgetsAccountId) {
+    linkProps.nearDevGovGigsWidgetsAccountId =
+      props.nearDevGovGigsWidgetsAccountId;
+  }
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+  const linkPropsQuery = Object.entries(linkProps)
+    .map(([key, value]) => `${key}=${value}`)
+    .join("&");
+  return `#/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+    linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+/* END_INCLUDE: "common.jsx" */
+
+const Scroll = styled.div`
+   {
+    z-index: -1;
+    margin-top: calc(-24px + 100px + 25px + 200px);
+  }
+`;
+
+if (!props.overviewId) {
+  return;
+}
+const overviewPost = Near.view(nearDevGovGigsContractAccountId, "get_post", {
+  post_id: Number(props.overviewId),
+});
+if (!overviewPost) {
+  return <div>Loading ...</div>;
+}
+
+if (!props.eventsId) {
+  return;
+}
+const eventsPost = Near.view(nearDevGovGigsContractAccountId, "get_post", {
+  post_id: Number(props.eventsId),
+});
+if (!eventsPost) {
+  return <div>Loading ...</div>;
+}
+
+initState({
+  tab: "Overview",
+});
+
+function switchTab(tab) {
+  State.update({ tab });
+}
+
+const discussionsRequiredLabels = ["community", props.label];
+const sponsorshipRequiredLabels = ["funding-funded", props.label];
```

Comment:
@ori-near How do you feel about being a bit optimistic and include ideas that did not get the tokens distributed to them? I.e. filter by "funding" instead of "funding-funded"?

Reply:
Hi @frol – sure that works. We can revisit this if needed.

Reply:
changed to funding as discussed with Ori